### PR TITLE
docs: Add missing files for the 2.33 sysadmin guide

### PR DESCRIPTION
**Pull request by jaboto**
The guide was missing some files.
Also added the SMS reporting one (related to Android App and the gateways that is already present in master and 2.34 -PR-)

### CODE REVIEW

File: src/commonmark/en/content/sysadmin/audit.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# Audit
```
Comment (Philip-Larsen-Donnelly) [inline src/commonmark/en/content/sysadmin/audit.md:1]: I don't think this audit service is applicable to 2.33.

File: src/commonmark/en/content/sysadmin/monitoring.md [hunk 2]
```diff
@@ -0,0 +1,411 @@
+# Monitoring
```
Comment (Philip-Larsen-Donnelly) [inline src/commonmark/en/content/sysadmin/monitoring.md:1]: I don't think the monitoring service is applicable to 2.33

File: src/commonmark/en/dhis2_system_administration_guide_INDEX.md [hunk 3]
```diff
@@ -5,3 +5,6 @@ title: 'DHIS 2 System Administration guide'
 
 !INCLUDE "content/common/about-this-guide.md"
 !INCLUDE "content/sysadmin/installation.md"
+!INCLUDE "content/sysadmin/monitoring.md"
```
Comment (Philip-Larsen-Donnelly) [inline src/commonmark/en/dhis2_system_administration_guide_INDEX.md]: monitoring not applicable to 2.33

File: src/commonmark/en/dhis2_system_administration_guide_INDEX.md [hunk 4]
```diff
@@ -5,3 +5,6 @@ title: 'DHIS 2 System Administration guide'
...
+!INCLUDE "content/sysadmin/audit.md"
```
Comment (Philip-Larsen-Donnelly) [inline src/commonmark/en/dhis2_system_administration_guide_INDEX.md]: audit not applicable to 2.33

**Review by Philip-Larsen-Donnelly — CHANGES_REQUESTED** [review]
Monitoring and Audit were introduced in 2.34

**Comment by jaboto** [comment]
Sorry about that. I copied the file from the other branch... Fixed now

**Pull request merged**